webui/react/src/themes.ts

```diff
@@ -153,7 +153,7 @@ export const lightTheme: Theme = {
     states: {
       active: '#009bde',
       failed: '#cc0000',
-      free: '#f7f7f7',
+      free: '#ececec',
```

Comment:
**nit:** how about going white instead now that the bg is darker? #ececec makes the wording a bit harder to read

Reply:
hmm the bg isn't non-white everywhere eg the resourcepool card. I'd have to style it differently if we wanna go with white. I can make it a bit bright